test/rules/std_L048_test.py

```diff
@@ -0,0 +1,66 @@
+"""Tests the python routines within L048."""
+import pytest
+
+import sqlfluff
+
+
+def test__rules__std_L048() -> None:
+    """L0048 is not raised for quoted literals surrounded by a single whitespace."""
+    sql = "SELECT a + 'b' + 'c' FROM tbl;"
+    result = sqlfluff.lint(sql)
+
+    assert len(result) == 1
+
+    results_l048 = [r for r in result if r["code"] == "L048"]
+    assert not len(results_l048)
```

Comment:
Think this is covered with the previous YML test cases so don't see need for it in Python.